### PR TITLE
Added new bundles

### PR DESCRIPTION
**Pull request by w3scout**

### CODE REVIEW

**Comment by Toflar** [comment]
Conflicts should be resolved here

**Comment by w3scout** [comment]
> Conflicts should be resolved here
Done!

File: meta/w3scout/contao-dompdf-bundle/de.yml [hunk 1]
```diff
@@ -5,4 +5,4 @@ de:
     keywords:
     - contao
     - bundle
-    - pdf
```
Comment (Toflar) [inline meta/w3scout/contao-dompdf-bundle/de.yml]: This change is not related. You don't need to delete ending new lines.

File: meta/w3scout/contao-dompdf-bundle/en.yml [hunk 2]
```diff
@@ -5,4 +5,4 @@ de:
     keywords:
     - contao
     - bundle
-    - pdf
\ No newline at end of file
+    - pdf
```
Comment (Toflar) [inline meta/w3scout/contao-dompdf-bundle/en.yml]: Same here :)

Comment (w3scout) [reply meta/w3scout/contao-dompdf-bundle/en.yml]: Not sure, I understand the problem. But hoping, it´s fixed now?!

**Comment by Toflar** [comment]
Fixed it myself in https://github.com/contao/package-metadata/commit/ca36db14341c1926b1b9ef202001711704f536a1 (language keys were wrong too :-))
